### ISSUE TITLE
Document WWID format and location

### ISSUE THREAD

**Issue by uhthomas** [siderolabs/talos#6839]
## Bug Report

Install disk selector does not match WWIDs correctly.

### Description

The install disk selector is capable of matching WWIDs, but it appears that it in some cases, it doesn't

https://www.talos.dev/v1.3/reference/configuration/#installdiskselector

I imagine the selector assumes the WWID will be present in `/sys/block/<device>/wwid` always, but this does not seem to be the case. Despite that, the WWID is present in `/dev/disk/by-id` as `wwn-<wwid>`. 

I verified this with a live Arch Linux ISO. There is nothing in `/sys/block/<device>/wwid` for any devices. My desktop does have WWIDs, though. 

However, `/sys/block/<device>/device/wwid` _does_ exist? Conversely, on my desktop it does not. Instead, it is available under the path `/sys/block/<device>/device/<device>/wwid`. Very strange. The WWID for the device is also formatted very differently to `/dev/disk/by-id/`. It follows the form `naa.12345` instead.

I hope this makes sense, there's a lot of weird information here.

To recap:

| path                                      | machine 1                              | machine 2              |
|  ---------------------------------------- | -------------------------------------- | ---------------------- |
| /sys/block/:device/wwid                  | `eui.e8238fa6bf530001001b448b4566aa1a` | does not exist         | 
| /sys/block/:device/device/wwid           | does not exist                         | `naa.50026b7381885d08` |
| /sys/block/:device/device/:device/wwid  | `eui.e8238fa6bf530001001b448b4566aa1a` | does not exist         |

### Logs

Not sure what to provide.

### Environment

- Talos version: v1.3.4
- Kubernetes version: N/A
- Platform: linux x86_64

**Comment by uhthomas** [comment]
Okay, I apologise. I think Talos is handling this just fine.

Could I request that we change this to a documentation issue? It would be great to document the format must be `naa.50026b7381885d08`, not `wwn-0x50026b7381885d08` or `0x50026b7381885d08` as shown by `ls -l /dev/disk/by-id/`. It would also be nice to consistently find the WWID, as on some systems or devices it can be in a different place.
